src/js/tracks/text-track-settings.js

```diff
@@ -327,7 +455,20 @@ class TextTrackSettings extends Component {
   }
 
   /**
-   * Restore text track settings
+   * Sets all <select> elements to their default values.
+   *
+   * @method setDefaults
+   */
+  setDefaults() {
+    Obj.each(selectConfigs, config => {
```

Comment:
But it's apparent that this could be confusing, I'll add parens around `config`. 👍